superdesk/publish/subscribers.py

```diff
@@ -107,8 +107,9 @@ def get(self, req, lookup):
             req = ParsedRequest()
         if req.args and req.args.get("filter_condition"):
             filter_condition = json.loads(req.args.get("filter_condition"))
-            return ListCursor(self._get_subscribers_by_filter_condition(filter_condition))
-        return super().get_from_mongo(req=req, lookup=lookup)
+            return self.hideConfigField(self._get_subscribers_by_filter_condition(filter_condition), "secret_token")
+
+        return self.hideConfigField(list(super().get_from_mongo(req=req, lookup=lookup)), "secret_token")
```

Comment:
would be good to handle more fields, like `apikey` or `password`

Reply:
okay